plugins/woocommerce/includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php

```diff
@@ -197,10 +200,14 @@ public static function get_suggestions_api_data() {
 
 		// If the options have never been updated, or were updated over a week ago, queue update.
 		if ( empty( $data['updated'] ) || ( time() - WEEK_IN_SECONDS ) > $data['updated'] ) {
-			$next = WC()->queue()->get_next( 'woocommerce_update_marketplace_suggestions' );
+			$queue = WC()->queue();
+			$next  = $queue->get_next( 'woocommerce_update_marketplace_suggestions', array(), WC_Marketplace_Updater::JOBS_GROUP );
 			if ( ! $next ) {
-				WC()->queue()->cancel_all( 'woocommerce_update_marketplace_suggestions' );
-				WC()->queue()->schedule_single( time(), 'woocommerce_update_marketplace_suggestions' );
+				$queue->cancel_all( 'woocommerce_update_marketplace_suggestions', array(), WC_Marketplace_Updater::JOBS_GROUP );
+
+				( $queue instanceof QueueWithPrioritiesInterface )
+					? $queue->add_with_priority( 'woocommerce_update_marketplace_suggestions', array(), WC_Marketplace_Updater::JOBS_GROUP, ActionQueuePriority::HIGH )
```

Comment:
Just double checking, but this will potentially have 2 overlapped `woocommerce_update_marketplace_suggestions` actions at the same time since the group is different now, right?

I don't necessarily think that's an issue, but wanted to call it out just in case.

Reply:
Correct. Practically, the action might be executed twice after the update, but later it gets back to normal execution frequency.